deploy/operator-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Ok, is this required for the metrics feature of the operator-sdk? I don't recall why it's missing

Reply:
I mean the service

Reply:
Yes, this is for metrics. It creates by default together with deployment but it's unable to control it parameters there. This is reason why we create it separately.